### ISSUE TITLE
no more `curl` in minikube v0.15.0

### ISSUE THREAD

**Issue by sebgoa** [kubernetes/minikube#1059]
`curl` used to be available in the boot2docker image of minikube.
It now seems that the OS has changed in v0.15.0 and there is no more curl.

That is a major inconvenience.

**Comment by r2d4** [comment]
In order to try to keep the image small, we've tried to change some of the ways that you debug in your VM.  

It's not a great solution, and we haven't really documented it yet, but one idea is to use something like the CoreOS toolbox container https://github.com/coreos/toolbox which lets you bring your own debugging and admin tools.

We've added it to the path here, so you can access it in your VM
``` 
minikube ssh
$ toolbox
```

https://github.com/kubernetes/minikube/pull/1034


That being said, I think we're definitely open to adding something like `curl` back in, since its so useful (much more so than `wget` in my opinion)

**Comment by sebgoa** [comment]
toolbox is not in the path afaik.

```
$ minikube version
minikube version: v0.15.0
sebair: Desktop $ minikube ssh 
$ uname -a
Linux minikube 4.7.2 #1 SMP Wed Jan 11 19:57:27 UTC 2017 x86_64 GNU/Linux
$ which toolbox
$
```

**Comment by r2d4** [comment]
Sorry it looks like its only in the latest image,

`https://storage.googleapis.com/minikube/iso/minikube-v1.0.5.iso`

We should be doing a release in the new few days with this as the default.  To get right now, you can either recreate the minikube VM after running 
`minikube config set iso-url https://storage.googleapis.com/minikube/iso/minikube-v1.0.5.iso`

You can also grab the script itself https://github.com/coreos/toolbox/blob/master/toolbox

**Comment by sebgoa** [comment]
ok, that's a real bummer. curl inside minikube is really handy....for testing/dev but also teaching k8s.

**Comment by robertsdionne** [comment]
Also, no `git` is included. This means `gitRepo` volumes no longer work properly!

**Comment by aadu** [comment]
and `rsync`

**Comment by aaron-prindle** [comment]
Closing.  This is resolved with v1.0.6 of the iso.  This will become the default iso in the latest release which will be later today.
